src/com/goide/inspections/unresolved/GoUnusedFunctionInspection.java

```diff
@@ -28,21 +28,30 @@
 import com.intellij.codeInspection.LocalInspectionToolSession;
 import com.intellij.codeInspection.ProblemHighlightType;
 import com.intellij.codeInspection.ProblemsHolder;
+import com.intellij.codeInspection.ui.SingleCheckboxOptionsPanel;
 import com.intellij.openapi.util.TextRange;
+import com.intellij.openapi.util.text.StringUtil;
 import com.intellij.psi.PsiElement;
 import com.intellij.psi.search.searches.ReferencesSearch;
 import org.jetbrains.annotations.NotNull;
 
+import javax.swing.*;
+
 public class GoUnusedFunctionInspection extends GoInspectionBase {
+  @SuppressWarnings("WeakerAccess")
+  public boolean ignoreExportedFunctions;
```

Comment:
The inspection that IntelliJ has for this will flag it as unused. I'll try and see I can use the field as a private one. In the example I had to make the option work the field was public (but maybe it was used in a different place?).